### DISCUSSION TITLE
create aab file with air_sdk

### DISCUSSION THREAD

**Discussion by josquinet** [airsdk/Adobe-Runtime-Support#2641] [General]
I am creating my aab file with air sdk in animate (d air config for android). For this I use the sdk managr of android studio. I would like to know which android api level is best to use for compatibility and other issues:

33, 32, 31?

thank you

**Comment by teotigraphix** [comment]
The AIR SDK always contains release notes and a table with all the compiler configurations for each platform.

https://airsdk.harman.com/api/versions/50.2.2.5/release-notes/Release_Notes_AIR_SDK_50.2.2.pdf

For instance 50.2.2:

![image](https://user-images.githubusercontent.com/226417/236237603-b0df5b28-a648-42e9-b04f-73f34294d2f2.png)

**Reply by josquinet** [reply]
thank you very match
